index.js

```diff
@@ -1,25 +1,53 @@
 'use strict';
 
+const modules = [require('babel-plugin-transform-es2015-modules-commonjs'), {
+  strict: false,
+}];
+
+const defaultTargets = {
+  android: 30,
+  chrome: 35,
+  edge: 14,
+  explorer: 9,
+  firefox: 52,
+  safari: 8,
+  ucandroid: 1,
+};
+
+function buildTargets(options) {
+  return Object.assign(defaultTargets, options.additionalTargets);
```

Comment:
this mutates the defaults; i think we want `Object.assign({}, defaultTargets, options.additionalTargets)` here (or `return { ...defaultTargets, ...options.additionalTargets };`)